openforcefield/tests/test_molecule.py

```diff
@@ -219,6 +220,27 @@ def mini_drug_bank(xfail_mols=None, wip_mols=None):
 # TESTS
 #=============================================================================================
 
+
+def known_openmm_elements():
+    """
+    Fixture containing elements known to both OpenMM's Element class and
+    QCElemental.
+    """
+    from simtk.openmm.app import element
+    elements = [getattr(element, name) for name in dir(element) if (type(getattr(element, name)) == element.Element)]
+
+    # The below runs into a problem with deuterium (fails name assertion)
+    elements.remove(element.deuterium)
```

Comment:
Deuterated drugs are a pretty big deal. I don't think that molecular simulations would realistically show a difference between H and D, but we should keep track of the lack of deuterium handling in an Issue so we remember this is here.